internal/cortex/querier/queryrange/results_cache_test.go

```diff
@@ -36,6 +37,14 @@ var (
 		Query: "sum(container_memory_rss) by (namespace)",
 		Stats: "all",
 	}
+	parsedHistogramRequest = &PrometheusRequest{
+		Path:  "/api/v1/query_range",
+		Start: 1536673680 * 1e3,
+		End:   1536716898 * 1e3,
+		Step:  120 * 1e3,
+		Query: "fake_histogram",
```

Comment:
Nit: if the metric is `{"fake": "histogram"}` then perhaps this query should be `{fake="histogram"}`? This doesn't change anything, just to be correct :thinking: